lambdas/build-client-oauth-response/src/test/java/uk/gov/di/ipv/core/buildclientoauthresponse/BuildClientOauthResponseHandlerTest.java

```diff
@@ -127,6 +127,47 @@ void shouldReturn200OnSuccessfulOauthRequest()
         assertEquals("test-state", params.get(1).getValue());
     }
 
+    @Test
+    void shouldReturn200OnSuccessfulOauthRequest_withNullIpvSessionAndClientSessionIdInHeader()
```

Comment:
Do we need another test to cover the situation where we have neither the ipvsession or clientsession and throw:
`throw new HttpResponseExceptionWithErrorBody(
                        HttpStatus.SC_BAD_REQUEST, ErrorResponse.MISSING_SESSION_ID);`
Or is this already covered by other tests?

Reply:
Just seen sonarcloud 100% coverage so can ignore this

Reply:
Done